source/documentation/quickstart/quickstart-guide.html.md

```diff
@@ -49,43 +43,50 @@ The following requirements are typical for small- to medium-sized installations.
 *   Static IP addresses for the oVirt Engine server and for each host server.
 *   DNS service that can resolve (forward and reverse) all the IP addresses.
 *   An existing DHCP server that can allocate network addresses for the virtual machines.
+*   If you need to use the storage from your hosts instead of using storage provided by SAN / NAS
+    consider moving to [oVirt-Gluster Hyperconvergence Guide](/documentation/gluster-hyperconverged/Gluster_Hyperconverged_Guide/)
 
 #### Virtual Machines
 
 Installation images for creating virtual machines, depending on which operating system you wish to use.
-
-*   Microsoft Windows XP, Vista, 7, 8, 2003, 2008 or 2012.
-*   Red Hat Enterprise Linux 5.x or 6.x.
-*   CentOS 6.x
-*   Fedora 16-20
-*   Ubuntu 12.04+
-*   openSUSE 12.x+
+A list of supported guest distribution is available in our [Download](/download/#supported-guest-distributions) page
```

Comment:
"A list of supported guest distributions is available on the [Download](/download/#supported-guest-distributions) page.